lib/slots.ts

```diff
@@ -17,11 +17,11 @@ export type GetSlots = {
   minimumBookingNotice: number;
 };
 
-const getMinuteOffset = (date: Dayjs, step: number) => {
```

Comment:
I have renamed `step` to `frequence` for better clarity, same as `getMinuteOffset(inviteeDate, frequency)`